### PR TITLE
TC 00.002

### PR DESCRIPTION
**Pull request by Sky8Ann**
https://trello.com/c/zyOrZNqt
https://trello.com/c/tVRtPWWG

### CODE REVIEW

**Review by w3code — APPROVED** [review]

File: src/test/java/CreatePipelineTest123.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+public class CreatePipelineTest123 extends BaseTest {
+
+@Test
+
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:10]: There should be no empty lines between annotation and the following method

File: src/test/java/CreatePipelineTest123.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    getDriver().findElement(By.xpath("//a[text()='Dashboard']")).click();
+
+    String actualPipelineName = getDriver().findElement
+            (By.xpath("//a[@class='jenkins-table__link model-link inside']")).getText();
+
+    Assert.assertEquals(actualPipelineName,expectedPipelineName);
+}
+
+@Test
+
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:32]: There should be no empty lines between annotation and the following method

File: src/test/java/CreatePipelineTest123.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+public class CreatePipelineTest123 extends BaseTest {
+
+@Test
+
+    public void testCreatePipeline123() {
+
+    String expectedPipelineName = "NewPipe";
+
+    getDriver().findElement(By.xpath("//span[text()='New Item']/../..")).click();
+    WebElement inputField = getDriver().findElement(By.xpath("//input[@name = 'name']"));
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:16]: Not the best choice for a variable name. It's unclear what element is stored in this variable. A better naming would be something like newJobNameInput

File: src/test/java/CreatePipelineTest123.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+    Assert.assertEquals(actualPipelineName,expectedPipelineName);
+}
+
+@Test
+
+    public void testCreatePipelinesWithTheSameName123 () {
+
+    getDriver().findElement(By.xpath("//span[text()='New Item']/../..")).click();
+    WebElement inputField1 = getDriver().findElement(By.xpath("//input[@name = 'name']"));
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:36]: Another bad variable name

File: src/test/java/CreatePipelineTest123.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+    getDriver().findElement
+            (By.xpath("//li[@class='org_jenkinsci_plugins_workflow_job_WorkflowJob']")).click();
+    getDriver().findElement(By.id("ok-button")).click();
+    getDriver().findElement(By.xpath("//button[@formNoValidate='formNoValidate']")).click();
+    getDriver().findElement(By.xpath("//a[text()='Dashboard']")).click();
+    getDriver().findElement(By.linkText("NewPipe")).isDisplayed();
+
+    getDriver().findElement(By.xpath("//span[text()='New Item']/../..")).click();
+    WebElement inputField2 = getDriver().findElement(By.xpath("//input[@name = 'name']"));
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:47]: Also not the best variable name

File: src/test/java/CreatePipelineTest123.java [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    inputField2.sendKeys("NewPipe");
+
+    String expectedErrorMassage = "» A job already exists with the name ‘NewPipe’";
+    String actualErrorMassage =
+            getDriver().findElement(By.xpath("//div[@class='input-validation-message']")).getText();
+
+    Assert.assertEquals(actualErrorMassage, expectedErrorMassage);
+
```
Comment (judmi) [inline src/test/java/CreatePipelineTest123.java:55]: You don't need a blank line here

**Review by SafronovaOlga — APPROVED** [review]

File: src/test/java/CreatePipelineTest123.java (hunk 6 above)
Comment (Sky8Ann) [reply src/test/java/CreatePipelineTest123.java:55]: before assert nor after?

**Pull request closed without merging**